core/src/avm2/object/xml_list_object.rs

```diff
@@ -135,6 +137,18 @@ impl<'gc> XmlListObject<'gc> {
         )
     }
 
+    pub fn to_xml_string(&self, activation: &mut Activation<'_, 'gc>) -> AvmString<'gc> {
```

Comment:
I guess clippy would like this better as `as_xml_string`...

Reply:
Should I rename this or allow the lint?

Reply:
I think renaming it is fine.

Reply:
Renamed.